app/core/src/main/java/com/fsck/k9/notification/NotificationChannelUtils.java

```diff
@@ -0,0 +1,113 @@
+package com.fsck.k9.notification;
+
+import android.app.NotificationChannel;
+import android.app.NotificationChannelGroup;
+import android.app.NotificationManager;
+import android.content.Context;
+import android.os.Build;
+import android.support.annotation.RequiresApi;
+
+import com.fsck.k9.Account;
+import com.fsck.k9.DI;
+import com.fsck.k9.Preferences;
+
+import java.util.HashMap;
+import java.util.List;
+
+public class NotificationChannelUtils {
+
+    public interface ChannelId {
+        String MESSAGES = "messages_channel";
+        String OTHER = "other_channel";
+    }
+
+    public static void updateChannels() {
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
+            return;
+        }
+
+        Context context = DI.get(Context.class);
```

Comment:
Please inject all dependencies via the constructor and make all class methods instance methods instead.